### ISSUE TITLE
Seamless Launch can't advance cards installed just before the start of turn window.

### ISSUE THREAD

**Issue by motionAlpha** [mtgred/netrunner#6972]
Cards installed after the runner finishes their turn, but before the corp clicks the 'Start Turn' button, are prevented from being advanced with Seamless Launch when they should be able to.

For example, installing an ice with Timely Public Release in Server 1, to trigger A Teia to install an agenda in Server 2 before the start of turn should allow the agenda to be advanced and scored with Seamless Launch

Link to Replay: https://www.jinteki.net/replay/eed2dec8-5dd8-41fb-a8a8-26ab7af2509a
Here the only card I can put the seamless counters on is the already installed card in server 1.

**Comment by NBKelly** [comment]
Yeah, currently you have to get your opponent to not click their "end turn" button until after you act.

I'll try to figure something out for this sooner or later, it's bugging me too :joy:

**Comment by the-all** [comment]
So woulnd't a workaround be to clear the installed this-turn marker when the corp (or in general, either player) presses "Start Turn", rather than at "End Turn"? :thinking:

**Comment by NBKelly** [comment]
As to that specific fix, see: https://github.com/mtgred/netrunner/pull/5924
Sooner or later when we actually implement paid ability windows, this should just get fixed "for free".
